BuildResidentialScheduleFile/resources/schedules.rb

```diff
@@ -846,7 +846,7 @@ def read_monthly_shift_minutes(resources_path:, daytype:)
   end
 
   def read_appliance_power_dist(resources_path:)
-    activity_names = ['clothes_washer', 'dishwasher', 'clothes_dryer', 'cooking']
+    activity_names = [Constants.ClothesWasher, Constants.Dishwasher, Constants.ClothesDryer, 'cooking']
```

Comment:
Why are some of these constants and some aren't?

Reply:
Good catch. These variables are internal to the algorithm (correspond to schedule resource files) and shouldn't be touched.